docs/guide/etl.rst

```diff
@@ -564,9 +564,10 @@ Extension
 ---------
```

Comment:
@fosskers what is going on here? Is this github flagging this section as invalid RST ?